### ISSUE TITLE
list TextStyle doesn't match any supplied TextStyles

### ISSUE THREAD

**Issue by KevinTheGray** [flutter/flutter_markdown#33]
I'm using a list element in a MarkdownBody, and the color/font size of the bullets/numbers don't match the "p" TextStyle, and  I don't see any way to modify the TextStyle of the bullets/numbers

**Comment by KevinTheGray** [comment]
The code here seems to indicate that it uses whatever style `new Text(...)` uses
```
Widget _buildBullet(String listTag) {
    if (listTag == 'ul')
      return const Text('•', textAlign: TextAlign.center);

    final int index = _blocks.last.nextListIndex;
    return new Padding(
      padding: const EdgeInsets.only(right: 5.0),
      child: new Text('${index + 1}.', textAlign: TextAlign.right),
    );
  }
```

**Comment by KevinTheGray** [comment]
thanks for the response.  I'll do a PR.  i'll try to get it in tonight

**Comment by KevinTheGray** [comment]
ok @DaveShuckerow, added the recommended changes and also updated the pubspec and changelog.  the PR is ready to be reviewed

**Comment by DaveShuckerow** [comment]
Fixed in #34
